patches/chrome-android-java-AndroidManifest_monochrome.xml.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/AndroidManifest_monochrome.xml b/chrome/android/java/AndroidManifest_monochrome.xml
+index f168f66b901ca39e63cbc617da766c52557510f9..2187dcea226d1cb4ff54fed97184f249fd1eddd2 100644
+--- a/chrome/android/java/AndroidManifest_monochrome.xml
++++ b/chrome/android/java/AndroidManifest_monochrome.xml
+@@ -17,7 +17,7 @@
+ 
+ {% block extra_application_attributes %}
+ {{ super() }}
+-android:multiArch="true"
```

Comment:
why is our bundle different from upstream?

Reply:
After PR https://github.com/brave/brave-core/pull/20849 we supply only single ABI libs, this was done to reduce the bundle size.

On Android 15 was added a more strict verification, https://android.googlesource.com/platform/frameworks/base/+/a55ab0340e16baa56e79c22688c349cc327a3361 and with the default `android:multiArch="true"` app refuses to be installed with an error
```
Error deriving application ABI
            com.android.server.pm.PackageManagerException: The multiArch app's native libs don't support all the natively supported ABIs of the device.
            	at com.android.server.pm.PackageAbiHelperImpl.maybeThrowExceptionForMultiArchCopy(PackageAbiHelperImpl.java:144)
            	at com.android.server.pm.PackageAbiHelperImpl.derivePackageAbi(PackageAbiHelperImpl.java:442)
            	at com.android.server.pm.InstallPackageHelper.preparePackageLI(InstallPackageHelper.java:1667)
            	at com.android.server.pm.InstallPackageHelper.installPackagesLI(InstallPackageHelper.java:1031)
            	at com.android.server.pm.InstallPackageHelper.installPackagesTraced(InstallPackageHelper.java:992)
``` 

Current Nightly/Beta/Stable work fine because the `targetSdkVersion=34` (Android 14), and starting from `cr128` it was changed to `35` (Android 15)

I'll update the message for c6809dda00888fe6815e213b4ccd58372df7ddd1 with addition
```
Related Chromium change:
https://source.chromium.org/chromium/chromium/src/+/75c9571291eba719e4d66ee0516a7b1f328dfba8

          Android V: rolling DEPS entries to start using Android V by default
          
          Bug: [341901237](https://issuetracker.google.com/341901237)
          Change-Id: [Iec05a0a95a4892dd7dbe4b18e3f0c158f679c79c](https://chromium-review.googlesource.com/q/Iec05a0a95a4892dd7dbe4b18e3f0c158f679c79c)
          Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5649225
```
I'll update after active review phase to avoid commits hash changing after the forced push.

Reply:
updated at https://github.com/brave/brave-core/pull/24227/commits/e36a73e3fa4522f74b0b97a8bb1d5140b521590e